.github/workflows/release.yml

```diff
@@ -70,13 +42,14 @@ jobs:
       - name: Deploy artifacts
         id: deploy
         if: github.ref == 'refs/heads/main'
-        run: ./deploy.sh
+        run: |
+          ./gradlew publishToSonatype closeAndReleaseStagingRepository --stacktrace
```

Comment:
Can we use the gradle-build-action here too?

Reply:
Coming to think of it, I don't see any value of using a custom action just to run commands that we can easily do with `./gradlew`. As a matter of fact, it just adds unnecessary maintenance. What do you think?

Reply:
Well, we could also use ./gradlew, it should work too. But we should align it, not mix both. Since we use the gradle-build-action already in many GitHub repos and workflows, we should then remove it everywhere.

The gradle-build-action is from the official gradle team on GitHub, it is not just some custom hackish action form a random single dude on the internet. They also document on their page why one should use it:
https://github.com/gradle/gradle-build-action#why-use-the-gradle-build-action

The maintenance is mostly done by Dependabot, so not much work for us. To reduce the number of PRs we get from Dependabot to update this action, we could even switch from the SHA1 hash to the version tag, so then we would only have to update that every leap year or so.